kuma/dashboards/templates/dashboards/includes/revision_dashboard_body.html

```diff
@@ -39,23 +38,46 @@
                         <a href="{{ revision_url }}">{{ revision.title }}</a>
                         <span class="dashboard-slug">{{ revision.document.slug }}</span>
                     </td>
-                    <td class="dashboard-comment">{{ format_comment(revision) }}</td>
-                    <td class="dashboard-author"><a href="{{ revision.creator.get_absolute_url() }}">{{ revision.creator }}</a>
-                    {% if show_ips and revision.revisionip_set.all() %}
-                    {% set revision_ip = revision.revisionip_set.all()[0].ip %}
-                    <span class="revision_ip">
-                    <br/>IP: {{ revision_ip }}
-                    {% set ban_ip_url = '%s?%s' % (url('admin:core_ipban_add'), 'ip=%s' % revision_ip) %}
-                    <br/><a class="dashboard-ban-ip-link" href="{{ ban_ip_url }}" target="_blank">{{ _('Ban IP from Editing') }}</a>
-                    </span>
-                    {% endif %}
-                    {% if request.user.is_superuser and revision.creator.active_ban %}
-                        <div class="banned">
-                            banned {{ datetimeformat(revision.creator.active_ban.date, format='date') }}
-                            by {{ revision.creator.active_ban.by }}
-                        </div>
-                    {% endif %}
+                    <td class="dashboard-comment">
+                        {{ format_comment(revision) }}
                     </td>
+                    <td class="dashboard-author">
+                        <a href="{{ revision.creator.get_absolute_url() }}">{{ revision.creator }}</a>
+                        {% set revision_ip = revision.revisionip_set.first() %}
+                        {% if show_ips and revision_ip %}
+                        <span class="revision_ip">
+                            <br/>IP: {{ revision_ip.ip }}
+                            {% if revision_ip.user_agent %}<br/>User agent: {{ revision_ip.user_agent }}{% endif %}
+                            {% if revision_ip.referrer %}<br/>User agent: {{ revision_ip.referrer }}{% endif %}
+                            {% set ban_ip_url = '%s?%s' % (url('admin:core_ipban_add'), 'ip=%s' % revision_ip) %}
+                            <br/><a class="dashboard-ban-ip-link" href="{{ ban_ip_url }}" target="_blank">{{ _('Ban IP from Editing') }}</a>
```

Comment:
``<br/>`` is good enough for me, looks clean in browser. A [description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) is a better semantic fit, but I bet you'd need CSS to make it as compact.